packages/selectable-list/lib/src/selectable-list/components/selectable-list/selectable-list.component.html

```diff
@@ -1,7 +1,21 @@
 <ul class="aui-selectable-list m-selectable-list m-selectable-list--no-border">
-    <li class="m-selectable-list__item" *ngFor="let item of items; let i=index;"  (click)="selectItem(item)" [ngClass]="i === index ? 'm-selectable-list__item--active' : ''">
-        <span *ngIf="!template && !itemTemplate" [innerHTML]="formatLabel(item)"></span>
-        <ng-template *ngIf="template" [ngTemplateOutlet]="template" [ngTemplateOutletContext]="{ item: item }"></ng-template>
-        <ng-template *ngIf="itemTemplate" [ngTemplateOutlet]="itemTemplate" [ngTemplateOutletContext]="{ $implicit: item }"></ng-template>
-    </li>
+	<li
+		class="m-selectable-list__item"
+		*ngFor="let item of items; let i = index"
+		attr.aria-selected="{{ i === index }}"
```

Comment:
isn't this always true? maybe i don't understand how ngFor works, but it seems that on each iteration it will increment index and set i to be equal to the incremented index.

Reply:
Nope, the first `index` is assigning `i` as the loop index. The others are refering to `index`, defined in the controller.